cmake/Modules/FindLibAJANTV2.cmake

```diff
@@ -32,6 +32,38 @@ find_path(
   PATHS /usr/include /usr/local/include /opt/local/include /sw/include
   PATH_SUFFIXES include)
 
+if(${_libajantv2_new_include_dir} STREQUAL "_libajantv2_new_include_dir-NOTFOUND")
+  find_path(
+    _libajantv2_old_include_dir
+    NAMES libajantv2
+    HINTS ENV
+          AJASDKPath${_lib_suffix}
+          ENV
+          AJASDKPath
+          ENV
+          DepsPath${_lib_suffix}
+          ENV
+          DepsPath
+          ${AJASDKPath${_lib_suffix}}
+          ${AJASDKPath}
+          ${DepsPath${_lib_suffix}}
+          ${DepsPath}
+          ${_AJA_NTV2_INCLUDE_DIRS}
+    PATHS /usr/include /usr/local/include /opt/local/include /sw/include
+    PATH_SUFFIXES include)
+  if(${_libajantv2_old_include_dir} STREQUAL "_libajantv2_old_include_dir-NOTFOUND")
+    set(AJA_LIBRARIES_INCLUDE_DIR ${_libajantv2_old_include_dir}/ajalibraries)
+    if(NOT LibAJANTV2_FIND_QUIETLY)
+      message(STATUS "aja: Using old ntv2 library")
```

Comment:
We should use deprecated if the functionality will be removed/obsoleted later. Given that this change is limited to the legacy build path, it will be removed one way or another.

Reply:
This same message and functionality is present in the new CMake finder as well.